stock_warehouse_flow/models/stock_warehouse_flow.py

```diff
@@ -10,6 +10,10 @@
 logger = logging.getLogger(__name__)
 
 
+class RollbackException(Exception):
```

Comment:
What about renaming it:

```suggestion
class ForceRollback(Exception):
```

While it can seems unconventional, I think `raise ForceRollback` is self explanatory.

Reply:
Indeed, I made the change